.gitignore

```diff
@@ -2,3 +2,5 @@ target/
 */target/
 */*/target/
 Cargo.lock
+.idea
```

Comment:
As a fellow IntelliJ user, I'd suggest solving this problem for yourself by putting the following in your `~/.gitconfig`:

```
[core]
  excludesfile = /home/myself/.gitignore
```

and then putting `.idea` in `~/.gitignore`